archstrike/burpsuite/PKGBUILD

```diff
@@ -3,7 +3,7 @@
 buildarch=1
 
 pkgname=burpsuite
-pkgver=1.7.19
+pkgver=1.7.26
```

Comment:
Hi. It looks like the newest version is 1.7.27. Once you change it, I think it should be good to merge.